src/rpc/RPCHelpers.cpp

```diff
@@ -1331,7 +1331,7 @@ parseBook(boost::json::object const& request)
     auto taker_pays = request.at("taker_pays").as_object();
     if (!taker_pays.contains("currency"))
         return Status{
-            RippledError::rpcINVALID_PARAMS, "missingTakerPaysCurrency"};
+            RippledError::rpcSRC_CUR_MALFORMED, "missingTakerPaysCurrency"};
```

Comment:
Is this the same error message rippled returns? I think rippled returns `"Missing field 'taker_pays.currency'."` But I could be reading the code wrong.